File: algorithms/math/LCM.js
// Function to return Least Common Multiple of two numbers 
function findLCM(a, b) {
    // Get the bigger number
    let larger = Math.max(a, b);
    // Get the smaller number
    let smaller = Math.min(a, b);

    // Add the larger number to itself till it is divisible by smaller number.
    // Return the number which is divisible.
    for (i = larger; ; i += larger) {
        if (i % smaller == 0)
            return i;
    }
}

let a = 21
let b = 4

console.log("LCM of " + a + " and " + b + " is " + findLCM(a, b));

File: Java/Questions/Number Questions/LCM.java
public class LCM {

    static int gcd(int a, int b) {
        int min = Math.min(a, b);

        for (int i = min; i >= 1; i--) {
            if (a % i == 0 && b % i == 0) {
                return i;
            }
        }
        return 1;
    }

    static int lcm(int a, int b) {
        return (a * b) / gcd(a, b);
    }

    public static void main(String[] args) {
        int a = 48;
        int b = 18;

        System.out.println(lcm(a, b));
    }
}